src/pages/api/sykmeldt/scenario/activescenario.ts

```diff
@@ -0,0 +1,22 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import {
+  activeTestScenario,
+  setActiveTestScenario,
+  TestScenario,
+} from "../../../../server/data/mock/activeTestScenario";
+import { displayTestScenarioSelector } from "../../../../environments/publicEnv";
+
+export default function handler(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
Burde vi ha en mekanisme for å ekskludere dette APIet i production environmentet?

Reply:
Ja, tenkte displayTestScenarioSelector holdt?

Reply:
Da må process.env["NEXT_PUBLIC_DISPLAY_TESTSCENARIO_SELECTOR"] === "true" stemme :)